sdk/storage/azure-storage-blob/tests/test_page_blob_async.py

```diff
@@ -1036,15 +1035,15 @@ def read(self, count):
             await blob.upload_page(data, offset=1024, length=513)
         except ValueError as e:
             self.assertEqual(str(e), 'length must be an integer that aligns with 512 page size')
-            pytest.skip("live only")
```

Comment:
Got catch - not sure why the "skip" was there.
Is there a reason this test doesn't use pytest.raises?